src/App.tsx

```diff
@@ -1,17 +1,139 @@
-import React from 'react';
-import './App.css';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import './App.scss';
+import classnames from 'classnames';
 
+import todos from './api/todos';
 import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoType } from './react-app-env';
+
+const preparedTodos: TodoType[] = todos.map(todo => (
+  {
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null,
+  }
+));
 
 const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [todosArr, setTodosArr] = useState(preparedTodos);
```

Comment:
Don't use data type names in variable names
```suggestion
  const [todos, setTodos] = useState(preparedTodos);
```